reference/release-notes.mdx

```diff
@@ -10,7 +10,46 @@ Full changelog can be viewed on our [GitHub Repo](https://github.com/architect-t
 
 <br/>
 
-# Breaking Change: Replace parameters with secrets keyword
+# April 2023 Breaking Change: Deprecation `path` and `port` for `liveness_probe`
+
+Starting from July of 2023, the usage of `path` and `port` for the `liveness_probe` will no longer be supported. Instead, the `command` option should be used for defining the liveness probe.
+
+## Who does this change affect?
+This change affects all users who are currently using the `path` and `port` options in their `liveness_probe` configuration. If you are one of them, please update your configuration to use the `command` option before July to avoid any disruptions in your application's health checks.
+
+## Why are we making this change?
+We are making this change to favor the `command` option, as it provides more flexibility in defining the liveness probe command as well as fixing an issue that could potentially arise if incorrect IP Address is passed to the `liveness_probe`.
+
+## What do you need to do?
+1. Remove all uses of `path` and `port` for the `liveness_probe` in your component configs.
+2. Add `command` for the `liveness_probe` and **re-register all components**. 
+
+The `liveness_probe` command specified in the `architect.yml` file will be executed periodically to determine the health of the container. If the command exits with a status code of 0, the container is considered healthy. However, if the command exits with a non-zero status code, the container is considered unhealthy and the health check will be retried according to the number of retries if specified. 
+
+## Examples
+Example: if you were using `path` and `port` in your component config as follow:
+```
+services:
+  app:
+    liveness_probe:
+      port: 3000
+      path: /
+```
+You would change this to a `curl` command, or a `wget` command. Whichever command you decide to use, make sure that it exists in your image.
```

Comment:
"You would change this to `curl` or `wget`, depending on which is installed on your image."